job-templates/kubernetes_master.json

```diff
@@ -5,14 +5,13 @@
       "orchestratorType": "Kubernetes",
       "orchestratorRelease": "1.17",
       "kubernetesConfig": {
-        "azureCNIURLLinux": "https://github.com/Azure/azure-container-networking/releases/download/v1.0.25/azure-vnet-cni-linux-amd64-v1.0.25.tgz",
-        "azureCNIURLWindows": "https://github.com/Azure/azure-container-networking/releases/download/v1.0.25/azure-vnet-cni-windows-amd64-v1.0.25.zip",
+        "azureCNIURLLinux": "https://github.com/Azure/azure-container-networking/releases/download/v1.0.25/azure-vnet-cni-linux-amd64-v1.0.28.tgz",
+        "azureCNIURLWindows": "https://github.com/Azure/azure-container-networking/releases/download/v1.0.25/azure-vnet-cni-windows-amd64-v1.0.28.zip",
```

Comment:
Can you bump up to 1.0.30? https://github.com/Azure/aks-engine/blob/3a2dcb2cb89fab9b05d50ff52a64e1d2155d82ac/pkg/api/const.go#L301

Reply:
updated